crates/jstz_cli/src/config.rs

```diff
@@ -242,7 +242,7 @@ struct Network {
 #[derive(Serialize, Deserialize, Debug, Clone, Default)]
 pub struct NetworkConfig {
     // if None, the users have to specify the network in the command
-    default_network: Option<NetworkName>,
+    pub default_network: Option<NetworkName>,
```

Comment:
If you use the `Config` methods, you should be able to avoid making this public

Reply:
done